tools/functional-tester/etcd-tester/tester.go

```diff
@@ -107,8 +106,8 @@ func (tt *tester) doRound(round int) (bool, error) {
 			plog.Printf("%s recovery error: %v", tt.logPrefix(), err)
 			return false, nil
 		}
-		plog.Printf("%s recovered failure", tt.logPrefix())
 
+		plog.Printf("%s recovered failure", tt.logPrefix())
```

Comment:
this should be moved back to below the `}` since that chunk of code concerns recovery and this message concludes the recovery phase